packages/@react-spectrum/menu/intl/en-US.json

```
@@ -0,0 +1,3 @@
+{
+    "more-actions": "More actions"
```

```suggestion
    "moreActions": "More actions"
```
just a nit pick, if you fix, I'll re-approve